resources/migrations/999_update_migrations.yaml.starter

```diff
@@ -0,0 +1,58 @@
+--------
+THIS IS THE STARTER MIGRATION FILE FOR NEW MAJOR VERSION OF METABASE
```

Comment:
I've never heard of a template file being called a "starter"... why not just call this a template?